pom.xml

```diff
@@ -128,6 +128,15 @@
     <buildDate>${maven.build.timestamp}</buildDate>
     <compileSource>1.8</compileSource>
     <java.min.version>${compileSource}</java.min.version>
+    <!-- override on command line to have generated LICENSE files include
+         diagnostic info for verifying notice requirements -->
+    <build.year>2022</build.year>
```

Comment:
we set this already via the `buildnumber-maven-plugin`. is that not working?

Reply:
I don't know why it's not working yet, but here is the output https://issues.apache.org/jira/secure/attachment/13040127/NOTICE.aggregate-no-build-year

```
Apache HBase Connectors - Assembly
Copyright 2018-${copyright-end-year} The Apache Software Foundation
```


let me continue to check it.

Reply:
I can't the reason -.- although I tried to use `-Dbuild.year=2022` and it worked.... I will revert the change for this build.year first. let me know if you find any thing but my goal is to have the dependency included included in the `NOTICE` file correctly.

Reply:
ping ?